src/main/java/com/shair13/data_service/dao/PageDetails.java

```diff
@@ -0,0 +1,15 @@
+package com.shair13.data_service.dao;
+
+public record PageDetails(
```

Comment:
To samo generalnie robi springowe Pageable. Brakuje mi tu obsługi sortowania w obu kierunkach (ascending/descending).